src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -2109,6 +2111,84 @@ private static void checkIntentForInboundAppLink(Context context) {
         }
     }
 
+    public void trackGestures(Activity parent) {
```

Comment:
`trackGestures` should not be on `MixpanelAPI`. Can you come up with a different file organization?